core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -80,12 +82,17 @@ private[spark] class TaskSetManager(
   val copiesRunning = new Array[Int](numTasks)
 
   val speculationEnabled = conf.get(SPECULATION_ENABLED)
+  private val speculationTaskProgressMultiplier = conf.get(SPECULATION_TASK_PROGRESS_MULTIPLIER)
+  private val speculationTaskDurationFactor = conf.get(SPECULATION_TASK_DURATION_FACTOR)
+  private val speculationTaskStatsCacheInterval = conf.get(SPECULATION_TASK_STATS_CACHE_DURATION)
```

Comment:
Shall we rename these confs to make it be specific to the task efffecency calculation feature? e.g., replace `speculation` with `efficient`.